src/components/Poster.js

```diff
@@ -0,0 +1,7 @@
+import '../css/Poster.css'
+
+const Poster = (props) => {
```

Comment:
Consider destructuring your props so you don't have to write `props.` so many times